backend/backend.go

```diff
@@ -646,17 +643,6 @@ func (backend *Backend) SystemOpen(url string) error {
 		}
 	}
 
-	if runtime.GOOS != "android" { // TODO: fix DownloadsDir() for android
```

Comment:
Does something change on Android?

Reply:
no, on android this branch was never executed.